README.md

```diff
@@ -423,13 +423,59 @@ for (const item of identities.items) {
 | `items` | [IEveryAuthIdentity](#ieveryauthidentity)[] | An array of EveryAuth credential objects. |
 | `next` | string \| undefined | If present, indicates that additional results may be obtained by supplying the `next` parameter to another call to `getIdentities` |
 
+#### deleteIdentity(serviceId, identityId)
+
+Deletes an existing identity.
+
+```javascript
+import everyauth from "@fusebit/everyauth-express";
+
+// Delete single identity
+const identityId = "idn-123"; // returned as part of getIdentitity and getIdentities
+await everyauth.deleteIdentity("slack", identityId);
+```
+
+##### Parameters  <!-- omit in toc -->
+
+| name | type | description |
+|------|------|-------------|
+| `serviceId` | string | The name of the remote service the user should be authenticated with. |
+| `identityId` | string | The identity ID returned as part of `getIdentity` or `getIdentities` call. |
+
+#### deleteIdentities(serviceId, identityId)
+
+Deletes all identities matching the search criteria.
+
+```javascript
+import everyauth from "@fusebit/everyauth-express";
+
+// Delete all identities tagged with tenantId 'contoso'
+const tenantId = "contoso"; 
+await everyauth.deleteIdentities("slack", { tenantId });
+```
+
+##### Parameters  <!-- omit in toc -->
+
+| name | type | description |
+|------|------|-------------|
+| `serviceId` | string | The name of the remote service to delete identities of. |
+| `idsOrTagsOrNull` | Record&lt;string,<br/>&nbsp;&nbsp;string&nbsp;\|<br/>&nbsp;&nbsp;number&nbsp;\|<br/>&nbsp;&nbsp;undefined&nbsp;\|<br/>&nbsp;&nbsp;null | A set of tags identities to be deleted must have. Specify `null` if you want to delete all identities of the service. |
+
 #### EveryAuthOptions
 
 | name | type | description |
 |------|------|-------------|
 | `finishedUrl` | string | The absolute or relative path to send the user to after completing the authorization flow. |
 | `mapToUserId` | async&nbsp;(req:&nbsp;[Express.request](https://expressjs.com/en/api.html#req))&nbsp;=>&nbsp;string | This method is called to generate a string user id to identify the user in your system and later allow querying EveryAuth for credentials owned by that user. |
 | `mapToTenantId` | async&nbsp;(req:&nbsp;[Express.request](https://expressjs.com/en/api.html#req))&nbsp;=>&nbsp;string | This method is called to generate a string tenant id to identify the tenant in your system, and allow querying EveryAuth for credentials owned by that tenant. If you don't specify this callback, the value of the tenant id will be set to the same value as the user id. |
+| `onCompleted` | async&nbsp;(req:&nbsp;[Express.request](https://expressjs.com/en/api.html#req), ctx:&nbsp;[IAuthorizedContext](#iauthorizedcontext))&nbsp;=>&nbsp;void | Called after a user successfully authorized to a target service but before their identity has been persisted. Perform any side-effect operations like removing prior identities that are no longer needed. |
```

Comment:
IEveryAuthContext, which has some overlap with an existing one, but...

All of the exported types follow an `IEveryAuth` prefix convention.

Reply:
Also, if `tags` is mutable, make that clear.  Consider including examples in code.